book_recommender_app/models.py

```diff
@@ -1,3 +1,10 @@
 from django.db import models
+from django.contrib.auth.models import AbstractUser
 
-# Create your models here.
+class App_User(AbstractUser):
+    email = models.EmailField(blank = False, null = False, unique = True)
+    USERNAME_FIELD = "email"
+    REQUIRED_FIELDS = []
```

Comment:
is username/email not required?

Reply:
The [Django docs](https://docs.djangoproject.com/en/4.2/topics/auth/customizing/) mentioned that the username field and password are required by default, so I left it blank!